### ISSUE TITLE
text-security doesn't work with exotic characters

### ISSUE THREAD

**Issue by edewit** [noppa/text-security#8]
example 'テ' or 'Б' will appear and not be replaced

**Comment by noppa** [comment]
This seems like a duplicate of #4.
Have you tried version 2 (alpha, not yet working in IE), as mentioned in [this comment](https://github.com/noppa/text-security/issues/4#issuecomment-445601943)?

**Comment by harterSe** [comment]
I had the same problem yesterday.
@noppa i did not se the version 2 branch :(

I made a fork of your project using typescript and addes the possibility to generate fonts with multiple unicode ranges. For example
```
npm run build -- -s disc square circle -u U+0-780 U+781-5555 U+5556-aaaa U+aaab-ffff
```
Will build 12 different fonts ( 3 shapes with 4 ranges ) and generate a css that combines them.
A modern browser will then only load the needed fonts.
https://github.com/harterSe/text-security

**Comment by edewit** [comment]
Solved by using version 2.0.0-alpha.2

**Comment by noppa** [comment]
@edewit @harterSe 

Just published version 3 has support for all unicode characters and for modern browsers the file size is actually smaller than it was in version 1. Also supports IE with a larger compatibility version of the font.
